Tasks/NuGet/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "nuget",
+  "version": "0.1.0",
+  "description": "Nuget Build Task",
+  "main": "nuget.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/vsts-tasks.git"
+  },
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/vsts-tasks/issues"
+  },
+  "homepage": "https://github.com/Microsoft/vsts-tasks#readme",
+  "dependencies": {
+    "vsts-task-lib": "^0.9.20"
```

Comment:
as part of v2 work i would recommend updating to use the new match pattern library so we can get to a consistent file match pattern experience across all in-the-box tasks